platform/machine/aws/machine.go

```diff
@@ -25,10 +25,12 @@ import (
 	"golang.org/x/crypto/ssh"
 
 	"github.com/coreos/mantle/platform"
+	m "github.com/coreos/mantle/platform/machine"
 	"github.com/coreos/mantle/util"
 )
 
 type machine struct {
+	*m.DefaultMachine
```

Comment:
Idea which I didn't validate: maybe a string could be used here and the `Board()` function added to this file. Same for other platforms.
Maybe this could spare the new DefaultMachine interface?

Reply:
I reverted the `DefaultMachine` interface and implemented a `Board()` for each `platform/machine/*/machine`. This one will return the `baseopts` from the current flight where the `Board` value has been passed from the `syncoptions`.